x-pack/legacy/plugins/infra/public/pages/metrics/components/sub_section.tsx

```diff
@@ -23,29 +23,25 @@ export const SubSection: FunctionComponent<SubSectionProps> = ({
   isLiveStreaming,
   stopLiveStreaming,
 }) => {
-  if (!children || !metrics) {
```

Comment:
:information_source: This condition violated the hook call order invariant so it was moved further down below the hook calls.